test_wrappers.py

```
@@ -4189,6 +4189,31 @@ def test_star_align(run):
+    )
+    run(
+        "bio/ngscheckmate/ncm",
+        ["snakemake", "--cores", "1", "fastq_paired_matched.txt", "--use-conda", "-F"],
+    )
+    
+    
+    
+
+
+
+@skip_if_not_modified
```

Ruff (0.7.0)

Undefined name `skip_if_not_modified`

(F821)